### PR TITLE
Keep folder structure of snippet binaries

### PR DESCRIPTION
**Pull request by joshuak94**

### CODE REVIEW

File: test/snippet/CMakeLists.txt [hunk 1]
```diff
@@ -51,7 +51,10 @@ macro (seqan3_snippet target_cpp snippet_full_path)
 
     add_executable (${target} ${source_path}/${target_cpp})
     target_link_libraries (${target} seqan3::test::unit)
-    # target_compile_options(${target} PUBLIC --disable-werror)
+    set_target_properties(${target}
+        PROPERTIES
+        RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${source_path}"
```
Comment (rrahn) [inline test/snippet/CMakeLists.txt]: Not sure about the source structure, but: `${CMAKE_BINARY_DIR}` is probably what you are looking for?
It is the mirrored structure of the cmake sources in the binary directory.

**Comment by joshuak94** [comment]
I'm not sure, Marcel wrote this on my computer. But when we tested it, it worked as expected, putting all the snippet executables in the proper subdirectories.

Josh

**Comment by rrahn** [comment]
Just talked to Marcel. All clear now.

**Comment by rrahn** [comment]
lgtm

**Pull request merged**